apsuite/commissioning_scripts/calc_tune_matrix.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python-sirius
```

Comment:
You should drop she-bang line in python import modules.

Reply:
ok, done.